lib/easel_provider.dart

```diff
@@ -609,7 +609,10 @@ class EaselProvider extends ChangeNotifier {
 
     disposePlayers(assetType: nft.assetType);
 
-    String residual = nft.tradePercentage.trim();
+
+
+    String tradePercentage = BigInt.from(int.parse(nft.tradePercentage.trim()) * kRoyalityPrecision).toString();
```

Comment:
"kRoyaltyPrecision" is a better spelling.

Reply:
Solved.